runtime/jnichk/jnicheck.c

```diff
@@ -548,21 +548,26 @@ jniCheckCallA(const char* function, JNIEnv* env, jobject receiver, UDATA methodT
 void
 jniCheckArgs(const char *function, int exceptionSafe, int criticalSafe, J9JniCheckLocalRefState *refTracking, const U_32 *descriptor, JNIEnv *env, ...)
 {
-	J9JavaVM *vm = ((J9VMThread*)env)->javaVM;
-	PORT_ACCESS_FROM_JAVAVM(vm);
+	J9JavaVM *vm = NULL;
 	va_list va;
 	const U_32 *code;
 	int argNum = 2;
-	UDATA trace = vm->checkJNIData.options & JNICHK_TRACE;
-	UDATA warn = (0 == (vm->checkJNIData.options & JNICHK_NOWARN));
-	J9VMThread * vmThread = (J9VMThread*)env;
+	UDATA trace;
+	UDATA warn;
+	J9VMThread *vmThread = (J9VMThread*)env;
+	J9VMThread *currentVMThread = globalJavaVM->internalVMFunctions->currentVMThread(globalJavaVM);
```

Comment:
This should be NULL checked and a different error output (thread not attached to the VM).

Reply:
updated in [47d194e](https://github.com/eclipse-openj9/openj9/commit/47d194ea2e0f95330bdcc673ddb0949d29409dac), I added a new error message for it in jnichk.nls